consensus/poet/core/tests/test_consensus/test_poet_block_publisher.py

```diff
@@ -88,11 +90,11 @@ def test_no_validator_registry(
 
         # create a mock_consensus_state that returns a mock with
         # the following settings:
-        mock_state = mock.Mock()
-        mock_state.validator_signup_was_committed_too_late.return_value = False
-        mock_state.validator_has_claimed_block_limit.return_value = False
-        mock_state.validator_is_claiming_too_early.return_value = False
-        mock_state.validator_is_claiming_too_frequently.return_value = False
+        mock_state = MockConsensusState().create_mock_consensus_state(
+            committed_too_late=False,
+            claimed_block_limit=False,
+            claiming_too_early=False,
+            claiming_too_frequently=False)
```

Comment:
The reason for the default parameters is so that you can do:
`mock_state = MockConsensusState.create_mock_consensus_state()`
(note you don't need () after `MockConsensusState`)
The only time you need to provide one of the keyword parameters is when you want to set it to `True`